CHANGELOG.md

```diff
@@ -5,66 +5,13 @@ All notable changes to this project are documented in this file.
-- Bumps `google-http-client-jackson2` from 1.35.0 to 1.42.3
-- Bumps `azure-core` from 1.33.0 to 1.34.0
-- Bumps `azure-core-http-netty` from 1.12.4 to 1.12.7
-- Bumps `maxmind-db` from 2.0.0 to 2.1.0
-- Bumps `json-schema-validator` from 1.0.69 to 1.0.73 ([#5316](https://github.com/opensearch-project/OpenSearch/pull/5316))
-- Bumps `proto-google-common-protos` from 2.8.0 to 2.10.0 ([#5318](https://github.com/opensearch-project/OpenSearch/pull/5318))
-- Bumps `protobuf-java` from 3.21.7 to 3.21.9 ([#5319](https://github.com/opensearch-project/OpenSearch/pull/5319))
-- Bumps `gradle-info-plugin` from 7.1.3 to 12.0.0 ([#5684](https://github.com/opensearch-project/OpenSearch/pull/5684))
-- Bumps `reactor-netty` from 1.0.18 to 1.1.1 ([#5685](https://github.com/opensearch-project/OpenSearch/pull/5685))
-- Bumps `apache-rat` from 0.13 to 0.15 ([#5686](https://github.com/opensearch-project/OpenSearch/pull/5686))
-
 ### Changed
-- Change http code for DecommissioningFailedException from 500 to 400 ([#5283](https://github.com/opensearch-project/OpenSearch/pull/5283))
-- Pre conditions check before updating weighted routing metadata ([#4955](https://github.com/opensearch-project/OpenSearch/pull/4955))
-- Integrate remote segment store in the failover flow ([#5579](https://github.com/opensearch-project/OpenSearch/pull/5579))
-- Gracefully handle concurrent zone decommission action ([#5542](https://github.com/opensearch-project/OpenSearch/pull/5542))
-
 ### Deprecated
-- Refactor fuzziness interface on query builders ([#5433](https://github.com/opensearch-project/OpenSearch/pull/5433))
-
 ### Removed
 ### Fixed
-- Fix 1.x compatibility bug with stored Tasks ([#5412](https://github.com/opensearch-project/OpenSearch/pull/5412))
-- Fix case sensitivity for wildcard queries ([#5462](https://github.com/opensearch-project/OpenSearch/pull/5462))
-- Support OpenSSL Provider with default Netty allocator ([#5499](https://github.com/opensearch-project/OpenSearch/pull/5499))
-- Apply cluster manager throttling settings during bootstrap ([#5524](https://github.com/opensearch-project/OpenSearch/pull/5524))
-- Update thresholds map when cluster manager throttling setting is removed ([#5524](https://github.com/opensearch-project/OpenSearch/pull/5524))
-- Fix backward compatibility for static cluster manager throttling threshold setting ([#5633](https://github.com/opensearch-project/OpenSearch/pull/5633))
-- Fix index exclusion behavior in snapshot restore and clone APIs ([#5626](https://github.com/opensearch-project/OpenSearch/pull/5626))
-- Fix graph filter error in search ([#5665](https://github.com/opensearch-project/OpenSearch/pull/5665))
-- [Test] Renaming PIT tests to IT to fix intermittent test failures ([#5750](https://github.com/opensearch-project/OpenSearch/pull/5750))
 ### Security
 
-[Unreleased 2.x]: https://github.com/opensearch-project/OpenSearch/compare/2.4...2.x
+[Unreleased 2.5]: https://github.com/opensearch-project/OpenSearch/compare/2.4...2.5
```

Comment:
I think this should be:

```
https://github.com/opensearch-project/OpenSearch/compare/b8a8b6c4d7fc7a7e32eb2cb68ecad8057a4636ad...2.5
```

That's the diff between the release commit for 2.5 (the latest commit today) and the latest on the 2.5 branch. There are no commits in the diff currently, but any changes merged into the 2.5 branch would show up.

If we create a release tag then this could be:

```
https://github.com/opensearch-project/OpenSearch/compare/2.5.0...2.5
```

but that tag doesn't exist yet.